src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(calculator.result, number);
+
+    return calculator;
+  };
+
+  calculator.add = (a, b) => a + b;
+
+  calculator.subtract = (a, b) => a - b;
+
+  calculator.multiply = (a, b) => a * b;
+
+  calculator.divide = (a, b) => a / b;
```

Comment:
The `divide` method should modify the calculator's internal `result` instead of taking two arguments. Additionally, handle division by zero by checking if `a` is zero and returning an appropriate message or handling it accordingly.